bridgesync/downloader.go

```diff
@@ -313,28 +319,28 @@ func buildRemoveLegacyTokenHandler(contract *bridgel2sovereignchain.Bridgel2sove
 	}
 }
 
-type call struct {
+type Call struct {
```

Comment:
This can be left unexported, WDYT?

```suggestion
type call struct {
```

Reply:
The same, is required in test code